src/components/Events/UserEvents/style.scss

```
@@ -173,3 +172,93 @@
     }
   }
 }
+
+.event-grid {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px; /* Spacing between cards */
+  justify-content: flex-start;
+  padding: 20px 0;
+}
+
+.event-card {
+  width: calc(25% - 20px); /* Ensure each card takes up 25% of the container width, minus the gap */
+  background: #fff;
+  border-radius: 10px;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.1);
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  transition: transform 0.2s;
+}
+
+.event-card:hover {
+  transform: translateY(-4px);
+}
+
+.cover {
+  height: 140px;
+  background-size: cover;
+  background-position: center;
+  position: relative;
+}
+
+.points-container {
+  position: absolute;
+  top: 8px;
+  right: 8px;
+  background: rgba(255, 255, 255, 0.9);
+  padding: 4px 8px;
+  border-radius: 20px;
+  font-size: 0.75rem;
+  font-weight: bold;
+}
+
+.attended-container {
+  position: absolute;
+  bottom: 8px;
+  left: 8px;
+  background: rgba(0, 128, 0, 0.8);
+  color: white;
+  padding: 4px 8px;
+  border-radius: 20px;
+  font-size: 0.75rem;
+}
+
+.content {
+  padding: 12px 15px;
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+}
+
+.event-card h2 {
+  font-size: 1rem;
+  font-weight: bold;
+  margin: 0 0 5px;
+}
+
+.event-card .location,
+.event-card .committee {
+  font-size: 0.85rem;
+  color: #555;
+  margin: 0;
+}
+
+.event-link {
+  margin-top: 10px;
+  align-self: flex-start;
+}
+
+/* Responsive layout */
+@media (max-width: 1024px) {
+  .event-card {
+    width: calc(50% - 20px); /* 2 cards per row */
+  }
+}
+
+@media (max-width: 600px) {
+  .event-card {
+    width: 100%; /* 1 card per row */
+  }
```

```suggestion
  .event-grid {
    justify-content: center;
  }
```
Centers the event cards on mobile view